lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -137,5 +137,26 @@ svg.nx-radio-checkbox__control {
         stroke: var(--nx-swatch-blue-40);
       }
     }
+
+    &:disabled{
+      + .nx-radio-checkbox__control {
+        &.nx-checkbox__box {
+          background-color:var(--nx-color-interactive-background-disabled);
+          border-color: var(--nx-color-interactive-border-disabled);
+
+          .fa-check {
+            color: var(--nx-color-interactive-border-disabled);
```

Comment:
Probably more correct to use `--nx-color-text-disabled` (though they are the same color)

Reply:
> Probably more correct to use `--nx-color-text-disabled` (though they are the same color)

Replaced `--nx-color-interactive-border-disabled `with `--nx-color-text-disabled`
[a429dc9](https://github.com/sonatype/sonatype-react-shared-components/commit/a429dc9e51134c98d0bff427f35ff5056e85151d)